CMakeLists.txt

```diff
@@ -404,7 +404,7 @@ IF( WIN32 )
         ${QtQuick_location}
         ${QtPositioning_location}
    )
-   IF( NOT ${NO_QTMULTIMEDIA})
+   IF(${NO_QTMULTIMEDIA})
       SET( Qt_DLLs,
         ${Qt_DLLs}
         ${QtMultimedia_location}
```

Comment:
Thanks Phillip. You're completely right. I got bitten by the double negative. There is still an error because the original version does not seem to work but I'll revert that change.